projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/StaticRoute.java

```diff
@@ -105,8 +99,8 @@ public int hashCode() {
     int result = 1;
     result = prime * result + _network.hashCode();
     result = prime * result + _administrativeCost;
-    result = prime * result + ((_nextHopInterface == null) ? 0 : _nextHopInterface.hashCode());
-    result = prime * result + ((_nextHopIp == null) ? 0 : _nextHopIp.hashCode());
+    result = prime * result + _nextHopInterface.hashCode();
```

Comment:
Hash codes are only valid within the context of a single JVM instance, so I don't think this can possibly affect anything.